app/src/main/java/ar/uba/fi/mercadolibre/model/ArticleBase.java

```diff
@@ -0,0 +1,32 @@
+package ar.uba.fi.mercadolibre.model;
+
+import java.util.List;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class ArticleBase {
+
+    @SerializedName("data")
+    @Expose
+    private List<Article> data = null;
+    @SerializedName("ok")
+    @Expose
+    private Boolean ok;
+
+    public List<Article> getData() {
+        return data;
+    }
+
+    public void setData(List<Article> data) {
+        this.data = data;
+    }
+
+    public Boolean getOk() {
```

Comment:
estos metodos se usan?

Reply:
Los getters/setters fueron todos autogenerados

Reply:
veo que sobrevivio el 'set data', eso tiene caso de uso en una clase llamada 'response'?

Reply:
exacto, un global search sobre el código te lo revela

Reply:
no usages found in all places me revela el ide

tal vez se confunde... de todos modos no entiendo la semantica de un setter sobre un response si es que realmente se usa un metodo de este tipo

Reply:
mergeo porque lo quiero usar ahora, despues de todo es solo un par de lineas de codigo muerto